lib/IO/CountDownInput.hpp

```diff
@@ -23,6 +23,7 @@ class CountDownEventStream : public EventStream {
 private:
   int n;              ///< How many events to generate
   const int interval; ///< Interval in ms between events
+  std::chrono::system_clock::time_point evTime;
```

Comment:
Store the interval as a std::chrono::milliseconds and document the evTime member.